### PR TITLE
Add basic support for GE Galaxy Pulsar Plus Controllers

### PR DESCRIPTION
**Pull request by craig-nokia**
This controller is used in the GE Infinity M Power Systems.

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/polling/os/gepulsar.inc.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<?php
```
Comment (Jellyfrog) [inline includes/polling/os/gepulsar.inc.php:1]: Could you please move this file to the new format, see https://docs.librenms.org/Developing/os/Initial-Detection/#os-discovery for more info

Comment (craig-nokia) [reply includes/polling/os/gepulsar.inc.php]: I just got to quit copying code and pay more attention to the documentation! LOL

Comment (Jellyfrog) [reply includes/polling/os/gepulsar.inc.php]: We changed this not too long ago, sorry for the inconvenience

Comment (craig-nokia) [reply includes/polling/os/gepulsar.inc.php]: Not a problem.  Also updated the snmpre and json test files so Travis will be happy.

File: LibreNMS/OS/Gepulsar.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+use LibreNMS\OS;
+
+class Gepulsar extends OS implements OSDiscovery
+{
+    public function discoverOS(): void
+    {
+        $device = $this->getDeviceModel();
+
+        $oids = [
+            'serial' => '.1.3.6.1.4.1.10520.2.1.3.2.12.0',
```
Comment (Jellyfrog) [inline LibreNMS/OS/Gepulsar.php:37]: cant we use named OIDS here instead?

Comment (craig-nokia) [reply LibreNMS/OS/Gepulsar.php:37]: I guess we could, I just looked a what is being done under LibreNMS/OS/*.php to match what I needed to do for the new format.  I noticed that very few are using the named oids.

**Review by Jellyfrog — APPROVED** [review]

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-68-release-changelog-september-2020/13525/1

**Pull request merged**